### PR TITLE
[MIG][14.0] document_page_project

### PR DESCRIPTION
**Pull request by mtelahun**
Migration to 14.0

### CODE REVIEW

File: document_page_project/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {
     "name": "Document Page Project",
     "summary": "This module links document pages to projects",
-    "version": "13.0.1.0.1",
+    "version": "14.0.1.0.1",
```
Comment (JoanMForgeFlow) [inline document_page_project/__manifest__.py:7]: 14.0.1.0.0, Minor change

**Review by JoanMForgeFlow — APPROVED** [review]
LGTM!

**Review by mariadforgeflow — APPROVED** [review]
LGTM. Can be merged when the requested change is done.

**Review by marcelsavegnago — APPROVED** [review]
LGTM

**Review by etobella — APPROVED** [review]

**Pull request merged**